gke/README.md

```diff
@@ -55,8 +67,16 @@ Instructions of the initial setup of the GCP project and GKE clusters for deploy
 
   * If you see authentication errors, need to contact DataCommons team to complete some of the role binding operations as the service account need access to the Cloud Bigtable and Big Query.
 
+* [Optional]: If creating a private instance of the graph powered by the Data Commons Mixer complete the following steps:
```

Comment:
This doc is meant to setup for the GKE instance.

I think we need a separate guide for starting a private mixer. Do you want to start a new doc under /docs/private_mixer.md and move this part there.

Reply:
Removed the optional steps for the private instance to be included in a separate document